src/AppNavigator.tsx

```diff
@@ -210,3 +205,44 @@ const useAutoLogout = () => {
     duration: 120 * 1000,
   })
 }
+
+const useOsAuthDisabled = () => {
+  const storage = useStorage()
+  const {authMethod} = useAuthMethod(storage)
+  const {canEnableOsAuth} = useCanEnableAuthOs()
```

Comment:
suspence, remove curly braces

Reply:
need it to refresh on focus in the application settings